config/exec_resource.go

```diff
@@ -55,6 +56,7 @@ func (e *execResource) download(ctx context.Context, execR *agentendpointpb.OSPo
 			} else {
 				name = "script"
 			}
+			perms = os.FileMode(0755)
```

Comment:
Should `OSPolicy_Resource_ExecResource_Exec_SHELL` set this too?

Reply:
No, fir SHELL we don't execute this file